meson.build

```diff
@@ -0,0 +1,36 @@
+project('sqlite3', 'c', license: 'pd', version: '3.34.1')
+
+cc = meson.get_compiler('c')
+libdl = cc.find_library('dl', required: false)
+libm = cc.find_library('m', required: false)
+
+sqlite_args = []
+if 'msvc' == cc.get_id() and 'static' != get_option('default_library')
+    sqlite_args += ['-DSQLITE_API=__declspec(dllexport)']
+endif
+
+includes = include_directories('.')
+libsqlite3 = library(
+    'sqlite3',
+    'sqlite3.c',
+    c_args: sqlite_args,
+    dependencies: [libdl, libm, dependency('threads')],
+    include_directories: includes,
```

Comment:
You don't need this for the compilation, current source and build dirs are always in header search path by default.